iteration.go

```diff
@@ -233,6 +262,39 @@ func createIterationLinks(request *goa.RequestData, id interface{}) *app.Generic
 	}
 }
 
+// ConvertFromLtreeFormat converts data to UUID format from ltree format.
+func ConvertFromLtreeFormat(uuid string) string {
+	// Ltree allows only "_" as a special character.
+	return strings.Replace(uuid, "_", "-", -1)
+}
+
+// IterationIDMap contains a map that will hold iteration's ID as its key
+type IterationIDMap map[uuid.UUID]*iteration.Iteration
+
+func parentPathResolver(itrMap IterationIDMap) IterationConvertFunc {
+	return func(request *goa.RequestData, itr *iteration.Iteration, appIteration *app.Iteration) {
+		parentUUIDStrings := strings.Split(iteration.ConvertFromLtreeFormat(itr.ParentPath), iteration.PathSepInService)
+		parentUUIDs := convertToUUID(parentUUIDStrings)
+		pathResolved := ""
+		for _, id := range parentUUIDs {
+			if i, ok := itrMap[id]; ok {
+				pathResolved += iteration.PathSepInService + i.Name
+			}
+		}
+		appIteration.Attributes.ResolvedParentPath = &pathResolved
+	}
+}
+
+func convertToUUID(uuidStrings []string) []uuid.UUID {
```

Comment:
thank you @hectorj2f  for you inputs, I am addressing this comment in another PR. yes those functions are copied from area.go because while building code both were PRs and wasn't sure on which one will make it to master first.
Strategy I discussed with @aslakknutsen  is to create `Path` as a Type and then we can build all related methods around it. So IMO this particular PR can go in with this. And I will work on Path as a Type.
Let me know what you think.